src/main/scala/scalafix/internal/sbt/ScalafixCompletions.scala

```diff
@@ -142,41 +127,43 @@ class ScalafixCompletions(
 
   def hide(p: P): P = p.examples()
 
-  def parser: Parser[Seq[String]] = {
+  def isSemantic(rule: String): Boolean =
```

Comment:
!

Reply:
Removed, it was not used. It was here originally as a dummy implementation before I could upgrade to the latest scalafix.